charts/latest/csi-driver-nfs/templates/csi-nfs-controller.yaml

```diff
@@ -31,10 +31,10 @@ spec:
       nodeSelector:
         kubernetes.io/os: linux
         {{- if .Values.controller.runOnMaster}}
-        node-role.kubernetes.io/master: ""
+        node-role.kubernetes.io/master: "true"
         {{- end}}
         {{- if .Values.controller.runOnControlPlane}}
-        node-role.kubernetes.io/control-plane: ""
+        node-role.kubernetes.io/control-plane: "true"
```

Comment:
not sure whether all master node are labeled with `node-role.kubernetes.io/control-plane: ""`, but as I know, the azure capz cluster are labeled as `node-role.kubernetes.io/control-plane: ""`

Reply:
I think you could set `set --nodeSelector=xxx` to make it work on your testing env if existing config does not work on your testing env.

Reply:
I can, but it's a footgun. Also, as I remember, at least GKE also has `"true"` instead of `""`.

Would you be open to a patch that replaces these node selector with node affinities (which make it possible to declare an "Exists" relation instead of a strict match)?